src/offline/wallet.ts

```diff
@@ -0,0 +1,88 @@
+import {fromSeed} from 'bip32'
+import {mnemonicToSeedSync} from 'bip39'
+import {decode, encodeForSigning} from 'ripple-binary-codec'
+import {deriveKeypair, generateSeed, verify} from 'ripple-keypairs'
+import {signOffline} from '../transaction/sign'
+import {SignOptions} from '../transaction/types'
+
+export interface WalletOptions {
+  // The entropy to use to generate the seed.
+  entropy?: Uint8Array | number[]
+
+  // The digital signature algorithm to generate an address for. Can be `ecdsa-secp256k1` (default) or `ed25519`.
+  algorithm?: 'ecdsa-secp256k1' | 'ed25519'
```

Comment:
Default should be `ed25519`, it's a newer algorithm and is more secure.

Reply:
Yes, I have the default set to `ed25519` on line 68: `options.algorithm = options.algorithm ?? Wallet.defaultAlgorithm`